### DISCUSSION TITLE
When I approve change it in grey

### DISCUSSION THREAD

**Discussion by UG-SEP** [MakeContributions/DSA#374] [Q&A] [answered]
Hey @ming-tsai,
A few days I saw that whenever I  approve, request changes its color  ✔ is grey.
Anyone knows why it shows grey color

**Comment by ming-tsai** [comment, marked as answer]
Yes, because your role was `Triage`, now your role is writer
![image](https://user-images.githubusercontent.com/37890026/123517212-c2a05980-d66d-11eb-9cba-9a0e99ea1f88.png)

**Reply by UG-SEP** [reply]
ohhh ok
